docs/src/remap/remap.adoc

```diff
@@ -1301,21 +1296,17 @@ def minimal_prolog(self, **words): # in remap module
 
 Return values:
 
-`INTERP_OK`::
-  Return this on success. You need to import this from `interpreter`.
+`INTERP_OK`:: Return this on success. You need to import this from `interpreter`.
 
-`"a message text"`::
-  Returning a string from a handler means 'this is an error message, abort the program'. Works like `(abort,` __<msg>__ `)`.
+`"a message text"`:: Returning a string from a handler means 'this is an error message, abort the program'. Works like `(abort,` __<msg>__ `)`.
```

Comment:
"a message text" or ``` `a message text` ```  but both seems a bit duplicated

Reply:
Changed to _a message text_:: . Have not rebased, yet.